.gitignore

```diff
@@ -0,0 +1,4 @@
+venv
+pycache
+.idea
+db.sqlite3
```

Comment:
Add empty line

It is a nit, but we should remember to do this always, so Github won't show us these red circles